### DISCUSSION TITLE
Question About Employer Requirements

### DISCUSSION THREAD

**Discussion by sforsyth089** [CMSgov/price-transparency-guide#257] [Q&A] [answered]
Hello - Does a self-funded Employer need to provide this information if all their health plans are meeting this requirement?

For example: does Employer A need to make the machine readable files if their administrator Health Plan A makes the files?

Perhaps this can be clarified within the README.
Thank you

**Comment by Devyandu** [comment, marked as answer]
Please see this discussion and reply from Scott (from CMS) which talks about a similar topic.
https://github.com/CMSgov/price-transparency-guide/discussions/36#discussioncomment-581547

Here is my understanding (I am not from CMS)
- Yes, it is the self-funded account's responsibility to post a link to the files on their public website
- They should be able to link to their relevant files which are on their plan administrator's site (rather than re-hosting them). 
- In this scenario, the self funded account is the **Payer** and the Plan Administrator is the **Reporting Entity** 
Hope this helps
Dev

**Reply by sforsyth089** [reply]
Thank you so much @Devyandu - very helpful !  Would love if CMS confirmed too.

**Reply by bryanbriegel** [reply]
Hi Stacy, this is noted in the regulations under special rules to avoid duplication - a self-funded group health plan sponsor may enter into a contract with a third party administrator to make the files available on their behalf.  (however if the administrator fails to comply the group health plan is deemed out of compliance.) "This special rule, as finalized, continues to permit a plan or issuer to satisfy the public disclosure
requirements of 26 CFR 54.9815 (/select-citation/2020/11/12/26-CFR-54.9815)-2715A3(b), 29 CFR
2590.715 (/select-citation/2020/11/12/29-CFR-2590.715)-2715A3(b), and, 45 CFR 147.212 (/selectcitation/
2020/11/12/45-CFR-147.212)(b) of the final rules by entering into a written agreement under which
another party (such as a TPA or health care claims clearinghouse) will make public the required information
in compliance with this section. The final rules identify TPAs and health care claims clearinghouses as
examples of the types of parties a plan or issuer may contract with, but these are not the only types of entities
that may enter into such arrangements and the Departments expect that they will comply with any applicable
privacy protection requirements, including applicable privacy protections under HIPAA.
Plans and issuers are not required to enter into such agreements in order to comply with the public
disclosure requirements of the final rules. As the Departments noted in the preamble to the proposed rules, if
a plan or issuer chooses to enter into such an agreement it is ultimately the responsibility of the plan or
issuer to ensure that the third party provides the information required by the final rules. As noted earlier in
this section, the special rule for insured plans is only available to plans that contract with an entity that is an
issuer separately subject to final rules. This requirement ensures that the Departments retain a mechanism
to enforce the final rules. Accordingly, this special rule relating to the use of third parties to satisfy these
requirements continues to provide that the plan or issuer would violate the requirements of the final rules if
the third party fails to provide full or timely information."

**Reply by sforsyth089** [reply]
Thank you Bryan. I'm reading this as an Employer does not have to do the work directly (can use a TPA/health care claims clearinghouse).  I'm wondering if the health plan itself counts as one of the other types of entities that would also be allowed.
